### ISSUE TITLE
OpenCensus spans cap number of attributes at 32

### ISSUE THREAD

**Issue by jdbennet2001** [census-instrumentation/opencensus-node#642]
Please answer these questions before submitting a bug report.

### What version of OpenCensus are you using?
0.0.16

### What version of Node are you using?
10.x

### What did you do?
If possible, provide a recipe for reproducing the error.
Generate a child span, and add more than 32 attributes
```
  let span     = tracer.startChildSpan({name, kind: 2});
  let i = 0;
  for ( ; i < 35; i++ ){ 
      span.addAttribute(`attr ${i}`, i);
 }
```

### What did you expect to see?
All attributes show up in Jaeger

### What did you see instead?
Only 32 attributes are preserved.

### Additional context
There appears to be a maximum number of attributes for each span, set here:
`https://github.com/census-instrumentation/opencensus-node/blob/b2726345ccb3b97e87bf29bb18c50c7486b1e560/packages/opencensus-core/src/trace/sampler/sampler.ts#L131`
Is there a reason why the maximum is set to 32?

**Comment by mayurkale22** [comment]
Just for your information, I am waiting on #644 and #647, If I don't get any update on those (may be till tomorrow) I will cut a new release.

**Comment by mayurkale22** [comment]
@jdbennet2001 Published in 0.0.17, let us know if you encounter any issues!
